docs/content/docs/integrations/aws.md

```diff
@@ -487,6 +492,10 @@ install_dependencies $LIB_PATH $ICEBERG_MAVEN_URL $ICEBERG_VERSION "${ICEBERG_PA
 install_dependencies $LIB_PATH $AWS_MAVEN_URL $AWS_SDK_VERSION "${AWS_PACKAGES[@]}"
 ```
 
+### AWS EKS
+
+[AWS Elastic Kubernetes Service (EKS)](https://aws.amazon.com/eks/) can be used to start any Spark, Flink, Hive, Presto or Trino clusters to work with Iceberg.
+Search the [Iceberg blogs](../../../blogs) page for tutorials around running Iceberg with Docker and Kubernetes.
```

Comment:
I've added the blog related to Docker that @samredai wrote in blog page, and we might also publish a few more blogs related to EKS usage, that's why I phrase it like this. But not sure if the number of `../` is correct, I still haven't figured out how to calculate the number of `../` to use. If @samredai you could provide some guideline it would be great.

Reply:
The relative link looks right. When the user is at this page they'll be at `/docs/<version>/aws` so `../../../blogs` gets them to `/blogs` which is right. You can test that out using this link: https://iceberg.apache.org/docs/latest/aws/../../../blogs

Reply:
Thanks for confirming!